framework/branch-key-store.md

```diff
@@ -109,6 +156,42 @@ If the [AWS KMS Configuration](#aws-kms-configuration) designates MRK ARN compat
 then two ARNs are compatible if they are equal in all parts other than the region.
 That is, they are compatible if [AWS KMS MRK Match for Decrypt](aws-kms/aws-kms-mrk-match-for-decrypt.md#implementation) returns true.
 
+If the [AWS KMS Configuration](#aws-kms-configuration) is Discovery or MRDiscovery,
+no comparison is ever made between ARNs.
+
+#### Discovery
+
+Discovery takes no additional information.
+
+The Keystore can use ANY KMS Key ARN already
+persisted to the backing DynamoDB table,
+provided they are in records created
+with an identical Logical Keystore Name.
```

Comment:
Issue/Suggestion:
(on behalf of @josecorella ): 
```suggestion
The Keystore MAY use the KMS Key ARNs already
persisted to the backing DynamoDB table,
provided they are in records created
with an identical Logical Keystore Name.
```

This change will need to propagated to many places.

Reply:
Resolved in 83ca2f1